src/frontend/src/meta_client.rs

```diff
@@ -230,7 +229,7 @@ impl FrontendMetaClient for FrontendMetaClientImpl {
     async fn list_version_deltas(&self) -> Result<Vec<HummockVersionDelta>> {
         // FIXME #8612: there can be lots of version deltas, so better to fetch them by pages and refactor `SysRowSeqScanExecutor` to yield multiple chunks.
         self.0
-            .list_version_deltas(0, u32::MAX, MAX_EPOCH)
+            .list_version_deltas(0, u32::MAX, u64::MAX)
```

Comment:
Let's use `HummockEpoch::MAX` as well.

Reply:
This pakage does not import `risingwave_hummock_sdk`.